src/dialog/overflow-menu/overflow-menu-custom-pane.component.ts

```diff
@@ -27,25 +27,26 @@ export class OverflowMenuCustomPane extends Dialog implements AfterViewInit {
 	constructor(
 		protected elementRef: ElementRef,
 		protected i18n: I18n,
+		protected animationFrameService: AnimationFrameService,
```

Comment:
_probably_ want to make this optional as well? I'm not sure how many people are extending the pane, but we should at least be consistent with `elementService` ... Applies to all the dialog.component derivatives